integration-tests/src/slowIntegrationTest/java/pl/allegro/tech/hermes/integrationtests/RemoteDatacenterProduceFallbackTest.java

```diff
@@ -127,14 +149,36 @@ public void shouldPublishAndConsumeViaRemoteDCWhenLocalKafkaIsUnavailable() {
                   .withLabels(
                       "group", topic.getName().getGroupName(),
                       "topic", topic.getName().getName(),
-                      "storageDc", REMOTE_DC_NAME)
+                      "storageDc", REMOTE_DC2)
                   .withValue(1.0);
               assertRemoteDCSendTotalMetric().withValueGreaterThan(remoteDCInitialSendTotal);
             });
   }
 
   @Test
-  public void shouldReturn500whenBothDCsAreUnavailable() {
+  public void shouldNotPublishViaRemoteDCNotListedInConfig() {
+    // given
+    TestSubscriber subscriber = subscribers.createSubscriber();
+    Topic topic =
+        initHelper.createTopic(
+            topicWithRandomName().withFallbackToRemoteDatacenterEnabled().build());
+    initHelper.createSubscription(
+        subscription(topic.getQualifiedName(), "subscription", subscriber.getEndpoint()).build());
+
+    // when both local and dc2 is not available, and dc3 is not listed in config
+    dc1.kafka.cutOffConnectionsBetweenBrokersAndClients();
+    dc2.kafka.cutOffConnectionsBetweenBrokersAndClients();
+
+    // and message is published
+    TestMessage message = TestMessage.of("key1", "value1");
+    DC1.publishUntilStatus(topic.getQualifiedName(), message.body(), 503);
+
+    // then no messages are received
+    subscriber.noMessagesReceived();
+  }
+
+  @Test
+  public void shouldReturn500whenAllDCsAreUnavailable() {
```

Comment:
this test is now essentialy the same as `shouldNotPublishViaRemoteDCNotListedInConfig` the one above, we should probably publish to DC2